projects/auth/src/lib/auth-form/auth-intern.component.html

```diff
@@ -1,13 +1,13 @@
 <form [formGroup]="form" role="form">
   <div>
     <mat-form-field class="full-width">
-      <input matInput required placeholder="{{'igo.auth.user' | translate}}" formControlName="username">
+      <input matInput required placeholder="{{'igo.auth.user' | translate}}" (keyup.enter)="loginUser(form.value)" formControlName="username">
```

Comment:
I think putting the "keyup.enter" on the form tag would be a little less verbose.
Or better, add a ngSubmit instead and put the login button with a type="submit"

Reply:
Done